content/docs/command-reference/add.md

```diff
@@ -144,9 +143,12 @@ directory symlinks.
 - `--external` - allow `targets` that are outside of the DVC repository. See
   [Managing External Data](/doc/user-guide/managing-external-data).
 
-- `--glob` - allows adding files and directories that match the specified
-  pattern as specified by `target`. Shell-style wildcards are supported: `*`,
-  `?`, `[seq]`, `[!seq]`, and `**`.
+  > Note that this option implies `--no-commit`, as external outputs are never
```

Comment:
hmm, not sure this is true/relevant here

Reply:
This was added after noticing people don't realize --external implies --no-commit but I don't have the links to those conversations right now. I only remember the last one (somewhere on Discord): basically a user asking how to backup external outputs.

So when(/if?) the import --store option gets merged, this will be revisited anyway. Up to you. I think that if it's not wrong, we should err on the side of TMI (too much info) in cmd refs.

Reply:
It looks wrong to me that the biggest issue. Unless I'm missing something.

Reply:
It's probably not a big issue. Thus it's a block quote note. So, should we remove it?

Reply:
It's kinda an issue if this information is wrong. I'm fine with both - remove and/or clarify and confirm.

Reply:
I guess you're right, external outputs can be cached, but externally. Is that what you meant? Sorry, just getting it now... Updated to:

  > Note that external outputs require an external cache setup (unless
  > `--no-commit` is used), and are never pushed or pulled from/to remote
  > storage. See link above for more details.